.python-version

```diff
@@ -0,0 +1 @@
+3.9.3
```

Comment:
This seems to be from pyenv and should be removed?

Reply:
Removed, thanks!